### ISSUE TITLE
Replace the use of the term ComponentDefinition in the GIU with Component or Part/Device

### ISSUE THREAD

**Issue by NeilWipat** [SynBioHub/synbiohub#363]
The user should not have to see terms arising from the SBOL data model - please replace them with human friendly terms

**Comment by cjmyers** [comment]
This is now down on dev.synbiohub.org, see for example:

https://dev.synbiohub.org/public/ACS_SynBio/GeneticToggle_collection/1

Note that I've changed the hyperlinks to point to wiki.synbiohub.org/Terms/SynBioHub to get definitions for these types.  These then point back to the SBOL specification.  I understand the concern with people pressing this link first, but I think this concern is outweighed by the need to have definitions for the terms.  However, since we are no longer using SBOL terms in all cases, a mapping of terms is needed which is why I added the wiki definitions.  Feel free to edit the language.
